src/app/qgslayertreeviewbadlayerindicator.cpp

```diff
@@ -82,7 +82,7 @@ void QgsLayerTreeViewBadLayerIndicatorProvider::onIndicatorClicked( const QModel
       QgsMessageViewer *m = new QgsMessageViewer( QgisApp::instance() );
       m->setWindowTitle( tr( "Layer Error" ) );
       if ( thisLayerErrors.count() == 1 )
-        m->setMessageAsPlainText( thisLayerErrors.at( 0 ) );
+        m->setMessageAsHtml( thisLayerErrors.at( 0 ) );
```

Comment:
Possibly not so nice for the existing uses of this indicator (e.g. the GDAL layer error messages). They might suddenly choke now if there's any < characters present...

Is HTML formatting really required here?